### DISCUSSION TITLE
Position of Operators in Variable Declaration

### DISCUSSION THREAD

**Discussion by keyseven123** [nebulastream/nebulastream#124] [Polls]
There are different positions to put the operator in a variable declaration.

**Comment by alepping** [comment]
Good arguments for `void* p`:
- Google C++ Style Guide: Prefers int* myInt, to clearly indicate that myInt is a pointer.
- Stroustrup's C++ Style: Bjarne Stroustrup, the creator of C++, prefers int* myInt, which is also reflected in his books.
